.github/workflows/create-portable-zip.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+import os
+import sys
+import zipfile
+
+import gha
+
+if len(sys.argv) != 3:
+    gha.print_error('Usage: create-portable-zip.py <output_path> <release|debug>')
+    sys.exit(1)
+else:
+    output_path = sys.argv[1]
+    configuration = sys.argv[2].lower()
+
+with zipfile.ZipFile(output_path, 'x', zipfile.ZIP_DEFLATED, compresslevel=9) as output:
+    output.mkdir('Extensions')
+    output.mkdir('Gallery')
+
+    output.write(f'artifacts/bin/Bonsai/{configuration}-repacked/Bonsai.exe', 'Bonsai.exe')
+    output.write(f'artifacts/bin/Bonsai/{configuration}/Bonsai32.exe', 'Bonsai32.exe')
+
+    nuget_config = [
```

Comment:
As mentioned during our meeting I hadn't noticed that file when I was working on this aspect. (Portable zip was done way before the installer.)

Thinking about this more, I wonder if the opposite might be the better strategy. Delete `Bonsai.Setup/Bonsai.config` and have the shipped NuGet.config come from Python.

I actually considered doing this in order to allow the installer to have a more appropriate NuGet.config corresponding to the CI job, but I didn't see most people wanting to use installers from unstable releases or testing forks. (Especially not until https://github.com/bonsai-rx/bonsai/issues/1849 happens since the `Bonsai.config` is hard to discover with the installer so nobody would see the instructions I wrote 😅)

Reply:
I was just going to go ahead and do this but it's getting late. I think it can wait for a separate PR if you like the idea. (I agree that messing with the XML DOM is a pain. We could make it a substitution template similar to what I did in the docs repo, but then we'd have this ugly substitution marker in our release NuGet.config files.)

Reply:
Created issue https://github.com/bonsai-rx/bonsai/issues/1881